book/website/collaboration/new-community/new-recruitment.md

```diff
@@ -0,0 +1,524 @@
+* race including colour, nationality, ethnic or national origin
+* religion or belief
+* sex
+* sexual orientation
+
+In some circumstances, you can use protected characteristics to help a disadvantaged or an underrepresented group during the recruitment process, for example by making reasonable adjustments for disability. This can help your organisation to be more diverse and representative. However, it is important to know what is possible and what could be against the law. Your action must not cause 'detriment' to another protected group (where someone is treated worse or discriminated against relative to someone else).
+
+Although you may want to make an impact to progress your team/organisation’s diversity, for example by  bringing more underrepresented groups from a STEM background into the organisation, it does not mean you can discount other candidates that meet the essential criteria, simply based on characteristics such as gender and/or age. This would be classed as unlawful discrimination. 
+
+Positive action can be used to encourage underrepresented groups to apply for roles at your organisation, however, there are limits to this and it’s recommended that you consult with your HR team before creating any initiatives. The UK Government has [guidance on their website](https://www.gov.uk/government/publications/positive-action-in-the-workplace-guidance-for-employers/positive-action-in-the-workplace) on the differences between positive discrimination and positive action.
+
+
+## Why aim for a diverse pool of job applicants?
+
+Building a diverse applicant pool and team isn't just the right thing to do morally and ethically; it's a strategic advantage that can drive business success and ensure work is carried out at the highest standards (see [Cheng and Zhange (2018)](https://www.mdpi.com/2071-1050/10/9/3134) for a review).
+
+Outlined below are business case arguments for incorporating practical strategies to attract diverse applicants into recruitment practices:  
+
+
+
+
+1. **Enhanced innovation and problem-solving** 
+Diverse teams bring together varied perspectives, experiences, and approaches, creating a possibility for more creative solutions and innovative ideas. 
+
+
+3. **Improved decision-making** 
+A diverse workforce helps mitigate unconscious bias, such as groupthink, resulting in more balanced and well-rounded decisions. For instance, a diverse data science team is more likely to recognise and address potential biases in datasets or machine learning models. 
+
+4. **Broader talent pool** 
+By taking concrete steps to cast a wider net, you increase your chances of finding exceptional talent that might otherwise be overlooked - particularly in competitive professions in which demand often outpaces supply. 
+
+5. **Enhanced company reputation** 
+Organisations known for their commitment to diversity often attract top talent across all demographics. 
```

Comment:
Could also mention that seeing other "people like me" in a company can give potential applicants more confidence to apply there. Not sure whether that fits here?